recipe_scrapers/quitoque.py

```diff
@@ -0,0 +1,92 @@
+from ._abstract import AbstractScraper
+from ._utils import get_minutes, get_yields, normalize_string
+
+
+class QuiToque(AbstractScraper):
+    @classmethod
+    def host(cls):
+        return "quitoque.fr"
+
+    @staticmethod
+    def _get_text(element):
+        if element:
+            return normalize_string(element.get_text())
+        else:
+            return None
+
+    def _get_time(self, time_name):
+        times = self.soup.select("div,.recipe-infos-short .item-info")
+        total_time = None
+        for time in times:
+            if time_name in time.get_text():
+                total_time = self._get_text(time).replace(time_name, "")
+        return get_minutes(total_time)
+
+    def _get_nutrients(self, nutrient_name):
+        nutrient_element = self._nutrient_list.find("p", string=nutrient_name).parent
+        return self._get_text(nutrient_element.find("p", class_="regular"))
+
+    def canonical_url(self):
+        return self.soup.find("meta", {"property": "og:url"}).get("content")
+
+    def author(self):
+        return "QuiToque"
+
+    def title(self):
+        return self._get_text(self.soup.find("h1", class_="title-2"))
+
+    def keywords(self):
+        product_tags = self.soup.find(id="product-tags").find_all(class_="badge")
+        keywords = {self._get_text(tag) for tag in product_tags}
+        return list(sorted(keywords))
+
+    def category(self):
+        category = self.soup.find(class_="primary-ghost")
+        return self._get_text(category)
+
+    def total_time(self):
+        return self._get_time("Total")
+
+    def prep_time(self):
+        return self._get_time("En cuisine")
```

Comment:
Is this `prep_time`, or should it be `cook_time`?

(prep = preparation; chopping vegetables, etc, cook = ingredients are being combined and/or changing chemical state - fried / oven-cooked, etc)

Reply:
I am pretty sure it is `prep_time`. "En cuisine" means "In kitchen" (but I'm pretty sure you already knew that), so it is the time you have to be in the kitchen to prepare the recipe. If you think it is still a mistake, please let me know.

Reply:
Yep, I think we interpret it the same way - it's how much time the recipe suggests would typically be required to be in the kitchen.. but I have a sense that that might include for both preparation and cooking steps.  The remaining time is idle / waiting for recipe steps to complete, perhaps?

In other words: my mental model is that `prep_time` always occurs before `cook_time`, but in this recipe I wonder whether `en cuisine` time contains some of both:

```
|------------------------------------------------|
| prep_time  |.................                  |
| cook_time  |                  .................|
| en cuisine |................. ....     ...   ..|
|------------------------------------------------|
```

(possibly; I'm not certain)

Reply:
Ok I see what you mean. I think you are partially right. Let's take [this recipe](https://www.quitoque.fr/products/cuisse-de-canard-au-romarin-et-pommes-de-terre-sarladaises-13720) as an example.
The indicated times are :
- Total: 1h50
- En cuisine: 50 min

So cooking time is 1 hour confirmed by the following instruction :

> Couvrez et enfournez 1h jusqu'à ce qu'elles soient cuites à coeur (la chair n'est plus rose).

> Cover and put in oven 1h until they are baked (flash is not pink).

But there also have some cooking time referenced in the recipe (like 5 or 10 minutes) not substracted from preparation time, because they often says "in the mean time prepare this".

Reply:
If you're comfortable with that, then I think the existing mapping is OK as-is :+1: 

If we have too much doubt, then we could omit `prep_time` and provide only `total_time` -- but I think it's useful to include both if we can (`cook_time` too, in theory, but it seems ambiguous here).